client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -179,7 +179,7 @@ export default connect(
 		let wpAdminUrl = get( site, 'options.admin_url' );
 		wpAdminUrl = wpAdminUrl
 			? formatUrl( {
-					...parseUrl( wpAdminUrl ),
+					...parseUrl( wpAdminUrl + 'admin.php' ),
```

Comment:
Suppose we can (but asking anyway); can we trust this is always without trailing slash?

I've noticed this being done like so elsewhere:
```js
	import { untrailingslashit } from 'lib/route';

	let adminURL = get( selectedSite, 'options.admin_url', '' );
	if ( adminURL ) {
		adminURL = untrailingslashit( adminURL ) + '/admin.php?page=jetpack';
	}
```

Reply:
There's quite a lot of precedent around the Calypso codebase that omits the slash (try searching for `admin_url + '`), so I'm assuming `admin_url` is guaranteed to always contain a trailing slash (or else we'd have seen a ton of bugreports by now :sweat_smile: ).

Reply:
Sounds good! :-)